docs/backend/guides/how-to/create-a-custom-linter.md

```diff
@@ -0,0 +1,137 @@
+# How to Create a Custom Go Linter
+
+Creating custom GO linters can be a great way to analyze your 
+project's source code and alert you to bugs, errors, 
+or other issues with your code.
+
+## Setting up your linter
+
+### File structure
+Start by setting up the files you'll need for your linter:
+```golang
+-cmd
+    -linter_name
+        -main.go
+
+-pkg
+    -linter_name
+        -linter.go
+        -linter_test.go
+```
+In the `cmd` folder create a folder for your linter called `<linter_name>` and add an empty file called `main.go`
+
+In `pkg` folder create another folder for your linter called `<linter_name>` and in it you will 
+have your `linter.go` and `linter_test.go`
+
+### Analyzer:
+In the `cmd` folder contains the the analysis driver for your linter. The file `main.go` will utilize a package called `singlechecker`,
+which defines the main function for an analysis driver and provides a tool to run a standalone analysis.
+
+The code in `main.go` is very simple:
+
+```golang
+package main
+
+import (
+	"example.org/linter_name"
+	"golang.org/x/tools/go/analysis/singlechecker"
+)
```

Comment:
Should this be an example that more closely matches what it would be like in our repo? E.g. something closer to 
```go
import (
        "golang.org/x/tools/go/analysis/singlechecker"

        examplelinter "github.com/transcom/mymove/pkg/example-linter"
)
```

Reply:
Yes, I think your example is much clearer